### DISCUSSION TITLE
Evaluating the Performance of Mathpix vs Deepdocdetection for Converting Scientific Papers

### DISCUSSION THREAD

**Discussion by yachty66** [deepdoctection/deepdoctection#153] [Q&A]
In my personal experience, I've found https://mathpix.com/ to be a highly effective tool for converting documents, particularly those involving complex scientific material. It seems to handle the intricacies of scientific papers quite well.

Has anyone in this community compared the performance between Deepdocdetection and Mathpix?

**Comment by JaMe76** [comment]
While I have not tested https://mathpix.com/, I will say with certainty that the extraction results are far superior to those of deepdoctection. The reason is this

Deepdoctection is not a product but a framework. It provides the boilerplate to develop such products (at least the backend) relatively quickly. However, it does not contain models that are designed to extract mathematical formulas or handwritten manuscripts. 
Depending on the use case, the corresponding models have to be trained or fine-tuned. 

Deepdoctection offers the possibility to save many thousands of lines of code. 

The pipelines that run with the public models are, if anything, only examples. Examples that may give reasonable results on data contained in the training corpus, but fail completely on documents based on a completely different distribution.
